src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateFromFormat = date.split(fromFormat[3]);
+  const dateToFormat = [];
+
+  for (let j = 0; j < 3; j++) {
+    for (let i = 0; i < 3; i++) {
```

Comment:
watch the video about algorithm complexity https://mate.academy/learn/algorithms/algorithm-complexity?section=video&videoId=660